UKHomeOffice/cerberus-service
# COP-10912 - Task details page new label

## Description
This PR simple adds the new label to a new task that is unclaimed.

## To Test
- Pull & run against dev
> Scenarios to test:
- For a task that is claimed (state is now **IN PROGRESS**), it **SHOULD NOT** display the **`NEW`** label in task details page.
- For a new task that is unclaimed, it **SHOULD** display the **`NEW`** label in task details page.
- For a task that is in **`ISSUED`** state, it **SHOULD NOT** display the **`NEW`** label in task details page.
- For a task that is in **`COMPLETE`** state, it **SHOULD NOT** display the **`NEW`** label in task details page.
